src/shared/components/post/post.tsx

```diff
@@ -124,6 +132,7 @@ interface PostState {
   maxCommentsShown: number;
   isIsomorphic: boolean;
   lastCreatedCommentId?: CommentId;
+  myUserInfoRes: RequestState<MyUserInfo>;
```

Comment:
Not necessary now, you can get `myUserInfo` from `this.isodata.myuserinfo` for top level routes.